### PR TITLE
Sign up page added

### PR DESCRIPTION
**Pull request by FirezerTe**
**-Index page modified**
-sign up page add

1. add
2. edit
3. fire

### CODE REVIEW

File: index.html
Comment (FirezerTe) [inline index.html:1]: Fix this.

File: signup.html
Comment (FirezerTe) [inline signup.html:1]: Add Form.

**Review by FirezerTe — COMMENTED** [review]
Fix those comments

**Review by Biyuktul — APPROVED** [review]
reviewed and approved

**Review by solasefiw — APPROVED** [review]
Approved

**Review by BerhanGit — APPROVED** [review]
Approved

**Pull request merged**